### ISSUE TITLE
Smart Shunt

### ISSUE THREAD

**Issue by KevinKTM** [KinDR007/VictronMPPT-ESPHOME#128]
This is the shunt I am using (headless) - not sure if this one is supported?

https://www.victronenergy.com/battery-monitors/smart-battery-shunt

- Verified ve.direct voltage is 3.3v
- Connected to pins 1 (ground) and 3 (tx) (slot at top) on the ve. ground and d7 on the d1 mini.
-  preferred setup is api -  I have not tried mqtt 

I started with the basic smartshunt-esp8266-example.yaml, commented out mqtt and enabled api (note ota requires a password now so samples should have this added). Also added captive_portal: Only entity reported was firmware version, no other sensors reported.

[logs_victron-smartshunt_logs.txt](https://github.com/KinDR007/VictronMPPT-ESPHOME/files/12555227/logs_victron-smartshunt_logs.txt)

**Comment by KevinKTM** [comment]
Here is the regular non-debug log. Looks like correct values are getting reported, I am not getting any sensors in HA (perhaps silly issue?)
[logs_victron-smartshunt_run.txt](https://github.com/KinDR007/VictronMPPT-ESPHOME/files/12555289/logs_victron-smartshunt_run.txt)

**Comment by KevinKTM** [comment]
Sorry, sorted my problem. When you create a default device in Esphome, an API key is added. I set a blank device up and obviously this key was associated with the device - the key is not in the sample template so after I added the key, all sensors appeared so now up and running! Friggin awesome. This is for testing for an HA instance I want to run on my RV and I think I will bypass getting a Cerbo as this gives me all the info I need!!!! So pumped!!! Thank you thank you.
